puppet/ha-puppet/screenshot.js

```diff
@@ -264,12 +303,47 @@ export class Browser {
             polling: 100,
           },
         );
+
+        // Wait for all images and Lovelace cards to load
+        await page.waitForFunction(
```

Comment:
let's move this to another PR. Also, let's put it inside the other function. No need to have 2 wait fors?